build/publish-mrt.yml

```diff
@@ -112,6 +112,7 @@ steps:
       native\MrtCore.C.props
       native\MrtCore.props
       native\MrtCore.targets
+      ProjectItemsSchema.xaml
       README.md
```

Comment:
>README.md [](http://example.com/codeflow?start=6&length=9)

this is probably a mistake

Reply:
Seems fine:
![image](https://user-images.githubusercontent.com/35987549/157728205-eb23721a-6093-4793-906c-aed6c63c610a.png)

I'll also verify the change with the NuGet package from the pipeline build.

Reply:
this is fine. we shouldn't have this file in the final package. but not your change. we can remove later.